src/cards/streak-card.js

```diff
@@ -0,0 +1,195 @@
+import Card from "../common/Card.js";
+import I18n from "../common/I18n.js";
+import { flexLayout, getCardColors, kFormatter } from "../common/utils.js";
+import { getStyles } from "../getStyles.js";
+import { streakCardLocales } from "../translations.js";
+
+const CARD_MIN_WIDTH = 300;
+const CARD_DEFAULT_WIDTH = 300;
+
+const createTextNode = ({
+  value,
+  label,
+  id,
+  index,
+  shiftValuePos,
+  bold,
+  number_format,
+}) => {
+  const kValue = number_format === "long" ? value : kFormatter(value);
+  const staggerDelay = (index + 3) * 150;
+
+  return `
+  <g class="stagger" style="animation-delay: ${staggerDelay}ms" transform="translate(25,0)">
+    <text class="stat ${bold ? "bold" : "not_bold"}" y="12.5">${label}:</text>
+    <text class="stat ${bold ? "bold" : "not_bold"}" x="${
+      shiftValuePos + 120
+    }" y="12.5" data-testid="${id}">${kValue}</text>
+  </g>
+  `;
+};
+
+const renderStreaksCard = (streaksData, options = {}) => {
```

Comment:
Here also a docstring with typings would be nice. See 

https://github.com/anuraghazra/github-readme-stats/blob/28b65928d2073c81b1d206e1197087e91d25778e/src/cards/stats-card.js#L79-L90